bwl_loot.json

```diff
@@ -560,7 +560,7 @@
   {
     "id": 19406,
     "name": "Drake Fang Talisman",
-    "role": "Warrior = Rogue = Hunter",
+    "role": "Warrior = Rogue = Hunter = Feral-Druid",
```

Comment:
IMO if we're letting Hunters bid we should let Ferals bid as well (arguably feral benefits the most from these because they have the hardest time getting hitcap and can be tanks)